### PR TITLE
added new dir structure for better organisation

### PR DESCRIPTION
**Pull request by bdbch**
This PR adds a more structured dir structure.

Main changes were the following folders:

* `utils.ts` became `./utils/index.ts` while `./utils/` also includes other utils files now
* `./components` is a folder containing all related component parts which are generic and can be reused again
* `./containers` is a folder containing components which are non-generic and have a specific logic (for example route views, the App component etc.)

I'd also think about an assets folder for the logo and screenshot images.

What are your thoughts?

### CODE REVIEW

**Comment by bdbch** [comment]
Just FYI I wasn't sure about the VBrowser dir yet. Shouldn't the folder (VBrowser) be called the same way the file/component is called? Shouldn't the component be called VBrowser instead of Video?

**Comment by howardchung** [comment]
I don't know if we need to distinguish between "containers" and "components". All JSX elements are supposed to be reusable, so App is really just a component?

Assets folder: Maybe could add in the future if we need more files in the public directory but not sure it needs to be done now.

Yeah Video could probably be renamed to VBrowser.

I was actually working on adding Firebase auth in my local dev environment, so there will probably be conflicting changes in one of those changesets.

**Comment by bdbch** [comment]
We could for sure put all of our elements into "components", usually I do this kind of separation to separate ui-only components from components handling data, states, user interaction, etc but it's a personal preference I guess.

should I do this?

What do you think would be a good start to migrate your changes into that pr? Do this one first?

**Comment by howardchung** [comment]
Yeah I think I like one directory of components. I think we can merge this first. I'm only touching App and TopBar and I think I can reapply my changes

**Comment by bdbch** [comment]
done! :) could be merged so we don't have a lot of issues later

File: src/components/routes/Settings/Settings.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 //@ts-ignore
-import { Button, Header, Modal, Form, TextArea, Icon } from 'semantic-ui-react';
+import { Button, Form, Header, Icon, Modal, TextArea } from 'semantic-ui-react';
```
Comment (howardchung) [inline src/components/routes/Settings/Settings.tsx:3]: minor note: Settings isn't really its own route, it's just a modal that can be rendered on any page. This is fine for now, can reorganize later. It's also not currently enabled.

**Pull request merged**